server/routes/claim.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
       failAction: async (request, h, error) => {
         console.log(request.payload)
         console.log(`rejected payload ${request.payload}`)
-        return h.response().code(400)
+        return h.response().code(400).takeover()

```

Comment:
Without the `takeover` Hapi complains that the wrong type has been returned from the failAction